hashTable/main.c

```diff
@@ -0,0 +1,39 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "HashTable.h"

```

Comment:
```suggestion
#include "hashTable.h"
```
Иначе под линуксом не работает